src/mongocxx/database.hpp

```diff
@@ -152,7 +152,7 @@ class MONGOCXX_API database {
     ///
     /// @return the name of this database.
     ///
-    stdx::string_view name() const;
+    bsoncxx::stdx::string_view name() const;
```

Comment:
I'm puzzled why this one can return a `string_view` but `collection::name` can't